### PR TITLE
module: Add EBS extra volume support

### PR DESCRIPTION
**Pull request by jieyu**
Allow extra EBS volumes to be created for each AWS instance.

### CODE REVIEW

**Review by fatz — CHANGES_REQUESTED** [review]
see dcos-terraform/terraform-aws-instance#6

File: main.tf [hunk 1]
```diff
@@ -121,6 +121,7 @@ module "dcos-infrastructure" {
   private_agents_os                          = "${var.private_agents_os}"
   private_agents_root_volume_size            = "${var.private_agents_root_volume_size}"
   private_agents_root_volume_type            = "${var.private_agents_root_volume_type}"
+  private_agents_extra_volumes               = ["${var.private_agents_extra_volumes}"]
```
Comment (fatz) [inline main.tf]: with this we need to have a minimum ref to the next version of instance

**Comment by fatz** [comment]
min version of infra needs to be raised as mentioned in dcos-terraform-terraform-aws-infrastructure#20 next patch num is safe to use

**Comment by jieyu** [comment]
@fatz PTAL again. I bump the dependency version

**Review by fatz — APPROVED** [review]
ci failed cause of a missing description update which is available in master. Approving and merging.

**Comment by rohithzr** [comment]
@fatz thanks, I have one question though, does this method also mounts those volumes to the instance?

**Pull request merged**